buildchow/templates/product-template.yaml

```diff
@@ -117,39 +109,44 @@
             - maven.test.error.ignore=true
             - download.cache.directory={download-cache}
             - targetPlatformGroup=jbdevstudio
-            - jbosstools-site=http://download.jboss.org/jbosstools/updates/nightly/core/${{jbosstools_site_stream}}/
+            - jbosstools-site=http://download.jboss.org/jbosstools/mars/snapshots/builds/jbosstools-build-sites.aggregate.site_${{jbosstools_site_stream}}/latest/all/repo/
           pom: ${{WORKSPACE}}/sources/product/site/pom.xml          
       - maven-target:
-          goals: "org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack"
-          maven-version: maven-3.x
+          goals: "-U org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack"
+          maven-version: maven-3.2.5
           private-repository: false
           properties:
-            - artifact=org.jboss.tools.releng:jbosstools-releng-publish:4.3.0.Alpha1-SNAPSHOT:zip
+            - artifact=org.jboss.tools.releng:jbosstools-releng-publish:4.3.0.CR1-SNAPSHOT:zip
```

Comment:
this seem to be referred multiple places - extract a value in `defaults.yaml` or in project definitions to reuse and ensure using the same consistently ?